phpseclib/Net/SFTP.php

```diff
@@ -1759,6 +1764,10 @@ function rmdir($dir)
      *
      * If $data is a resource then it'll be used as a resource instead.
      *
+     *
+     * Setting $mode to self::SOURCE_CALLBACK will use $data as callback function, which gets only one parameter -- number 
```

Comment:
Opps, sorry, my fault, thank you for report.
Fixed in another PR, commit 0cc6125f8795f577aa9e01535c1a58545e290866

AG

Andreas Fischer wrote:
>
> In phpseclib/Net/SFTP.php <https://github.com/phpseclib/phpseclib/pull/658#discussion_r29546820>:
>
> > @@ -1759,6 +1764,10 @@ function rmdir($dir)
> >       *
> >       * If $data is a resource then it'll be used as a resource instead.
> >       *
> > +     *
> > +     * Setting $mode to self::SOURCE_CALLBACK will use $data as callback function, which gets only one parameter -- number
>
> NET_SFTP_CALLBACK ?
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/phpseclib/phpseclib/pull/658/files#r29546820>.
>